app/_locales/en/messages.json

```
@@ -1043,7 +1043,7 @@
     "message": "Reset Account"
   },
   "resetAccountDescription": {
-    "message": "Resetting your account will clear your transaction history."
+    "message": "Resetting your account clears local settings, activity, and logs. This will not change the balances in your accounts."
```

This description isn't accurate though. This button does two things: It clears the transaction history for the currently selected account, and it resets network connections.

What this tends to be useful for is developers using test networks like `ganache`, so that they can reset the network without re-installing the extension. It has I believe also been used as a workaround for various bugs, but that's about it.

---

The old description was pretty decent, though the added part about how it won't change balances might still be worth adding.

Really the name of the button itself is misleading. "Clear Transaction History" might be more accurate. We could also clarify in the description that it only applies to the current selected account, not all accounts.

---

This was mostly copied from #7493, I'm open to suggestions about what the copy should be

---

I would suggest changing the button to say "Clear Transaction History", and the description to "This will clear your transaction history and reset network connections. This is most commonly used after resetting a local test network."

Except I'm not sure if "local test network" is the best way to describe that. It's the best I could think of 🤷‍♂

---

@dpazdan does this make sense to you? Does this resemble what people are asking about it for?

---

I think folks are really familiar with "Reset Account" (esp devs) so I would advocate for keeping that phrase as-is.

I like the warning. MOst non-dev users are most concerned about whether this will require them to re-set up their account, so maybe we should add `...this will not change the balances in your accounts or require you to re-enter your seed phrase`

---

Yea I'm good with whatever works for both devs and users.  The intent was primarily to help users understand they will not lose money in their accounts, or have to re-import their seed phrase/accounts.  If there is too much info to add, maybe we do a modal or add more information on the popup after clicking "reset account".  

Long term it would be really nice to have a list of items that can be reset broken out, and then a reset all option.

---

I suggested renaming the button because it was requested by devs (in #5577 ), and I believe the poor choice in name is the root cause for users being confused about what it does.

---

I'm good to keep the button as-is and adjust the wording to be the following:

```suggestion
    "message": "Resetting your account clears local settings, activity, and logs. This will not change the balances in your accounts or require you to re-enter your seed phrase."
```

I, personally, think the word "reset" here carries correct-_er_ connotations than "clear" and that the phrase "Clear History" doesn't suggest that it would have anything to do with settings/networking.

---

Since this is also a destructive action, we should use red buttons for the `reset button` on this screen and in the confirmation modal.

---

@cjeria updated in #7696